src/app/main/main.html

```diff
@@ -63,11 +63,11 @@
     </div>
   </div>
 
-  <modal id="bookmark-list" ng-if="Bookmarks.isSupported">
```

Comment:
have you tested the embedded mode (load in iframe with `sandbox="allow-scripts"`?